bindings/cs/MongoDB.Libmongocrypt/PinnedBinary.cs

```diff
@@ -24,6 +24,23 @@ namespace MongoDB.Libmongocrypt
     /// <seealso cref="Binary" />
     internal class PinnedBinary : Binary
     {
+        #region static
+        internal static void HandleAsPinnedBinary<THandle>(THandle handle, byte[] bytes, Status status, Func<THandle, BinarySafeHandle, bool> handleFunc) where THandle : CheckableSafeHandle
```

Comment:
minor: `Handle` naming is a bit unclear. But I don't have better suggestions, maybe `HandleCheck`?

Reply:
The right name here probably should be like: `RunWithBytesAsPinnedBinary`, maybe we can simplify it to: `RunAsPinnedBinary`?